docker/dockerfile

```
@@ -30,22 +36,22 @@ RUN [[ $(id -u ubuntu) = $USER_UID ]] || usermod -u $USER_UID ubuntu \
 
 # Install scanning secrets tools
-RUN curl -sSfL https://raw.githubusercontent.com/trufflesecurity/trufflehog/main/scripts/install.sh | sh -s -- -b /usr/local/bin
-RUN git clone https://github.com/gitleaks/gitleaks.git \
+RUN curl -sSfL https://raw.githubusercontent.com/trufflesecurity/trufflehog/v${TRUFFLEHOG_VERSION}/scripts/install.sh | sh -s -- -b /usr/local/bin
+RUN git clone -b v${GITLEAKS_VERSION} --depth 1 https://github.com/gitleaks/gitleaks.git \
   && cd gitleaks \
   && make build
 ENV PATH="${PATH}:/gitleaks"
 
-RUN sudo curl -o /usr/local/bin/jq -L https://github.com/stedolan/jq/releases/download/jq-1.8.1/jq-linux64 && sudo chmod +x /usr/local/bin/jq
+RUN sudo curl -o /usr/local/bin/jq -L https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 && sudo chmod +x /usr/local/bin/jq
```

Hadolint (2.14.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)